[RDS] `opentelekomcloud_rds_instance_v1` panics on read

<!--
Hi there, thank you for opening an issue.
-->


# Affected Resource(s)
- `opentelekomcloud_rds_instance_v1`

# Terraform Configuration Files
```hcl
data "opentelekomcloud_rds_flavors_v1" "flavor" {
    region = "eu-de"
    datastore_name = "PostgreSQL"
    datastore_version = "9.5.5"
    speccode = "rds.pg.s1.medium.ha"
}

resource opentelekomcloud_networking_secgroup_v2 sg {
  name = "sg-rds-test"
}

resource "opentelekomcloud_rds_instance_v1" "instance" {
  name = "rds-instance"
  datastore {
    type = "PostgreSQL"
    version = "9.5.5"
  }
  flavorref = data.opentelekomcloud_rds_flavors_v1.flavor.id
  volume {
    type = "COMMON"
    size = 100
  }
  region = "eu-de"
  availabilityzone = "eu-de-01"
  vpc = var.router_id
  nics {
    subnetid = var.network_id
  }
  securitygroup {
    id = opentelekomcloud_networking_secgroup_v2.sg.id
  }
  dbport = "8635"
  backupstrategy {
    starttime = "01:00:00"
    keepdays = 1
  }
  dbrtpd = "Blurppp!120521"
  ha {
    enable = true
    replicationmode = "async"
  }
  tag = {
    foo = "bar"
    key = "value"
  }
  depends_on = ["opentelekomcloud_networking_secgroup_v2.sg"]
}
```

# Debug Output/Panic Output
```
panic: dbport: '' expected type 'string', got unconvertible type 'int'

goroutine 467 [running]:
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*ResourceData).Set(0xc0006e5c80, {0x14737e2, 0x6}, {0x1216660, 0xc000b24f98})
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource_data.go:230 +0x2a5
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/rds.resourceInstanceRead({0xc00050d510, 0xc0008d4ab0}, 0xc0006e5c80, {0x1465940, 0xc000bb6000})
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/rds/resource_opentelekomcloud_rds_instance_v1.go:451 +0xff6
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/rds.resourceInstanceCreate({0x1704cb8, 0xc0010d11a0}, 0xc00050d7b8, {0x1465940, 0xc000bb6000})
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/rds/resource_opentelekomcloud_rds_instance_v1.go:382 +0xd5e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc00035e460, {0x1704c48, 0xc000c32500}, 0x2, {0x1465940, 0xc000bb6000})
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:330 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc00035e460, {0x1704c48, 0xc000c32500}, 0xc0010a9960, 0xc000c1d600, {0x1465940, 0xc000bb6000})
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:456 +0x871
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc000346108, {0x1704c48, 0xc000c32500}, 0xc000f1bea0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/grpc_provider.go:955 +0x9aa
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ApplyResourceChange(0xc0008d02c0, {0x1704cf0, 0xc000c2cea0}, 0x4ea)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:332 +0x6c
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x140f280, 0xc0008d02c0}, {0x1704cf0, 0xc000c2cea0}, 0xc0010d0d80, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00063a540, {0x171e010, 0xc000816780}, 0xc0006e0600, 0xc0007f4660, 0x205c280, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1194 +0xc8f
google.golang.org/grpc.(*Server).handleStream(0xc00063a540, {0x171e010, 0xc000816780}, 0xc0006e0600, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1517 +0xa2a
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x294


Process finished with the exit code 1
```

# Steps to Reproduce
<!--
Please list the steps required to reproduce the issue, for example:
-->
1. `terraform apply`

--- a/opentelekomcloud/services/rds/resource_opentelekomcloud_rds_instance_v1.go
+++ b/opentelekomcloud/services/rds/resource_opentelekomcloud_rds_instance_v1.go
@@ -3,6 +3,7 @@ package rds
 import (
 	"context"
 	"log"
+	"strconv"
 	"strings"
 	"time"
 
@@ -448,7 +449,7 @@ func resourceInstanceRead(_ context.Context, d *schema.ResourceData, meta interf
 			"[DEBUG] Error saving volume to Rds instance (%s): %s", d.Id(), err)
 	}
 
-	mErr = multierror.Append(mErr, d.Set("dbport", instance.DbPort))
+	mErr = multierror.Append(mErr, d.Set("dbport", strconv.Itoa(instance.DbPort)))
 
 	datastoreList := make([]map[string]interface{}, 0, 1)
 	datastore := map[string]interface{}{

--- a/releasenotes/notes/fix-rds-v1-015fcf5224d0d250.yaml
+++ b/releasenotes/notes/fix-rds-v1-015fcf5224d0d250.yaml
@@ -1,0 +1,4 @@
+---
+fixes:
+  - |
+    **[RDS]** Fix panic when using ``resource/opentelekomcloud_rds_instance_v1`` (`#1354 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1354>`_)

